ReactWindows/ReactNative.Shared/Bridge/ReactInstance.cs

```diff
@@ -33,6 +36,13 @@ class ReactInstance : IReactInstance
 
         private bool _initialized;
 
+        public event OnTransitionToBridgeHandler OnTransitionToBridgeHandler;
+
+        private bool BridgeBusy
```

Comment:
Why one event instead of two (one for busy, one for idle)?

Reply:
great question, I'm not sure why we deviated from Java there. I'll change it to be congruent with Java for easier back/forward porting of fixes/enhancements with upstream.

Reply:
#fixed